build/CMakeLists.txt

```diff
@@ -201,6 +202,49 @@ ExternalProject_Add(prometheus-cpp
     -DCMAKE_INSTALL_PREFIX:PATH=${CMAKE_CURRENT_BINARY_DIR}/prometheus-cpp/install
 )
 
+#
+# Build crc32c
+#
+ExternalProject_Add(crc32c
+  PREFIX crc32c
+  GIT_REPOSITORY "https://github.com/google/crc32c.git"
+  GIT_TAG "1.0.6"
+  SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}/crc32c/src/crc32c"
+  CMAKE_CACHE_ARGS
+    -DBUILD_SHARED_LIBS:STRING=yes
```

Comment:
Can this be built static and have it be built-into GCS library so we don't need to deliver it as separate shared library?